### PR TITLE
Robust CI testing for workflow creation

### LINKED ISSUE #12: Have `write_dax_to_dir` return the file path the dax was written to
Currently there is no easy way to add the dax path into the submit script generator without reconstructing it yourself. Having `wirte_dax_to_dir` return the dax path would allow it to be more easily used within code.

### LINKED ISSUE #14: Implement robust testing of a sample workflow in CI

### PR DESCRIPTION
**Pull request by LichMaster98**
Closes #14 and Closes #12

### CODE REVIEW

File: scripts/multiply_by_x.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+
+
+def main(params: Parameters):
+    input_file_path: Path = params.existing_file("input_file")
```
Comment (gabbard) [inline scripts/multiply_by_x.py:8]: nit: the type annotation should be unnecessary due to type inference.

File: scripts/multiply_by_x.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    output_file_path: Path = params.creatable_file("output_file")
+    x = params.integer("x")
+    with open(str(input_file_path)) as input_file:
```
Comment (gabbard) [inline scripts/multiply_by_x.py:11]: nit: you can do `with input_file_path.open() as input_file`

**Comment by LichMaster98** [comment]
I'm going to break out coverage of the `key_pair` to #21 because #20 contains changes to that section of the code

File: scripts/multiply_by_x.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
```
Comment (gabbard) [inline scripts/multiply_by_x.py:1]: Add a comment to this and the other test scripts that they are for testing purposes.

**Pull request merged**